apps/condo/pages/settings/index.tsx

```diff
@@ -47,16 +50,21 @@ const SettingsPage: React.FC = () => {
     const ControlRoomTitle = intl.formatMessage({ id: 'ControlRoom' })
     const PropertyScopeTitle = intl.formatMessage({ id: 'pages.condo.settings.propertyScope.title' })
     const EmployeeRolesTitle = intl.formatMessage({ id: 'EmployeeRoles' })
+    const MobileFeatureConfigTitle = intl.formatMessage({ id: 'pages.condo.settings.barItem.MobileFeatureConfig' })
 
     const hasSubscriptionFeature = hasFeature('subscription')
-
+    const { useFlag } = useFeatureFlags()
+    const hasMobileFeatureConfigurationFeature = useFlag(MOBILE_FEATURE_CONFIGURATION)
```

Comment:
Looks like I still have the ability to create a model using graphql API directly.

Reply:
this is true